projects/aws/eks-a-admin-image/Makefile

```diff
@@ -0,0 +1,124 @@
+-include .env
+
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG=0.0.0-dev # This value does not matter but needs to be set
+
+SIMPLE_CREATE_BINARIES=false
+HAS_LICENSES=false
+REPO_NO_CLONE=true
+IMAGE_NAMES=
+
+REPO=eks-a-admin-image
+REPO_OWNER=aws
+
+BUILD_TARGETS=validate
+RELEASE_TARGETS=build-raw-image
+
+PACKER_VERSION=1.8.0
+
+EKSA_VERSION?=v0.0.0
+EKSA_RELEASE_MANIFEST_URL?=https://dev-release-prod-pdx.s3.us-west-2.amazonaws.com/eks-a-release.yaml
+
+FULL_OUTPUT_DIR?=$(MAKE_ROOT)/$(OUTPUT_DIR)
+AMI_OUTPUT_DIR?=$(FULL_OUTPUT_DIR)/ami
+OVA_OUTPUT_DIR?=$(FULL_OUTPUT_DIR)/ova
+MANIFEST_FILE?=manifest.json
+
+ARTIFACTS_UPLOAD_PATH?=$(PROJECT_PATH)
+EXPORT_AMI_BUCKET?=$(ARTIFACTS_BUCKET)
+AMI_S3_DST=$(EXPORT_AMI_BUCKET)/$(ARTIFACTS_UPLOAD_PATH)
+EXPORT_AMI_DST=$(AMI_S3_DST)/$(EKSA_VERSION)/$(GIT_HASH)/
+LATEST_AMI_S3_URL=$(AMI_S3_DST)/$(LATEST)/snow-admin.raw
+
+EXPORT_OVA_BUCKET?=$(ARTIFACTS_BUCKET)
+EKSA_ADMIN_PROJECT_PATH?=projects/aws/eks-a-admin-image
+OVA_PROJECT_PATH?=$(EKSA_ADMIN_PROJECT_PATH)/ova
+
+DEPENDENCIES_FOLDER=$(MAKE_ROOT)/$(OUTPUT_DIR)/dependencies
+PACKER=$(DEPENDENCIES_FOLDER)/packer
+
+include $(BASE_DIRECTORY)/Common.mk
+
+build-%: export PKR_VAR_eks-a-version=$(EKSA_VERSION)
+build-%: export PKR_VAR_eks-a-release-manifest-url=$(EKSA_RELEASE_MANIFEST_URL)
+build-%: export PKR_VAR_build-version=$(shell echo $(GIT_HASH) | cut -c1-7)
+build-%: KUBECTL_URL?=$(shell ./build/get_kubectl_url.sh)
+build-%: export PKR_VAR_kubectl-url=$(KUBECTL_URL)
+
+$(PACKER): OS=$(shell uname -s | tr '[:upper:]' '[:lower:]')
+$(PACKER):
+	mkdir -p $(@D)
+	curl --silent -L https://releases.hashicorp.com/packer/$(PACKER_VERSION)/packer_$(PACKER_VERSION)_$(OS)_amd64.zip > packer.zip
+	unzip packer.zip -d $(@D)
+	rm packer.zip
+
+$(FULL_OUTPUT_DIR):
+	mkdir -p $(FULL_OUTPUT_DIR)
+
+$(AMI_OUTPUT_DIR): $(FULL_OUTPUT_DIR)
+	mkdir -p $(AMI_OUTPUT_DIR)
+
+$(OVA_OUTPUT_DIR): $(FULL_OUTPUT_DIR)
+	mkdir -p $(OVA_OUTPUT_DIR)
+
+.PHONY: build-raw-image
+build-raw-image: build-ami export-to-raw
+
+.PHONY: build-ami
+build-ami: export PKR_VAR_manifest-output=$(AMI_OUTPUT_DIR)/$(MANIFEST_FILE)
```

Comment:
nit: we could define something like `IMAGE_TYPE_OUTPUT_PATH` and define ` build-ami: export PKR_VAR_manifest-output=$(IMAGE_TYPE_OUTPUT_PATH)` in `build-%`